### PR TITLE
[Enhancement] Optimize log print and error message.

### LINKED ISSUE #11751: [Enhancement] Optimize log print and error message.
## Enhancement
Three are three improvements about log print and error message:

1. setup error msg in agent task return msg.
2. show backends state info, when fe can't find useful backends.
3. remove useless error msg when help zip file not found

### PR DESCRIPTION
**Pull request by luohaha**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：
Fixes #11751

## Problem Summary(Required) ：
In this pr, there are three part improvements:
1. setup error msg in agent task return msg.
2. show backends state info, when fe can't find useful backends.
3. remove useless error msg when help zip file not found

E.G, When frontend can't find useful backends, the error msg now can show the current state of backends, and you will find out why they can't be used.
```
mysql> select * from tbl;
ERROR 1064 (HY000): Backend not found. Check if any backend is down or not. backend: [11001-192.168.58.1-9660 alive: false inBlacklist: false] [11097-192.168.58.1-9661 alive: false inBlacklist: false] [11098-192.168.58.1-9662 alive: false inBlacklist: false]
```
## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

**Review by decster — DISMISSED** [review]

**Review by murphyatwork — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java [hunk 1]
```diff
@@ -1619,6 +1619,28 @@ private TNetworkAddress toBrpcHost(TNetworkAddress host) throws Exception {
...
+                infoStr += String.format("[%d-%s-%d alive: %b inBlacklist: %b] ", backendID, backend.getHost(), 
+                                    backend.getBePort(), backend.isAlive(), SimpleScheduler.isInBlacklist(backendID));
+            }
+            return infoStr;
+        } else {
+            String infoStr = "backend: ";
+            for (Map.Entry<Long, Backend> entry : this.idToBackend.entrySet()) {
+                Long backendID = entry.getKey();
+                Backend backend = entry.getValue();
+                infoStr += String.format("[%d-%s-%d alive: %b inBlacklist: %b] ", backendID, backend.getHost(),
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java:1637]: IP is enough. The backendID and port is not necessary

Comment (chaoyli) [reply fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java:1637]:
```
backend: xxx. error: xxx
```
is clear

Comment (luohaha) [reply fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java]: done

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 24 (29.17%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/sql/ast/RangePartitionDesc.java|0|1|00.00%|[134]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

**Review by decster — APPROVED** [review]

**Review by mark-phoenix-ai — APPROVED** [review]

**Review by meegoo — APPROVED** [review]

**Comment by luohaha** [comment]
https://github.com/Mergifyio backport branch-2.5

**Pull request merged**